docs: remove schedule

StephenFluin opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit


## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes


This should be merged to master, 9.0, and 8.x.

gkalpak commented on aio/content/guide/releases.md:
```diff
@@ -84,6 +90,7 @@ This cadence of releases gives you access to new features as soon as they are re
 
 
 
+<<<<<<< HEAD
```
😱

gkalpak commented on aio/content/guide/releases.md:
```diff
@@ -105,6 +112,8 @@ The following table contains our current target release dates for the next two m
 
 
+=======
+>>>>>>> docs: remove schedule
```
😱 
😱

gkalpak commented on aio/content/guide/releases.md:
```diff
@@ -84,27 +90,6 @@ This cadence of releases gives you access to new features as soon as they are re
 
 
 
-{@a schedule}
```
This was being linked to from a couple of places (so at least we should remove those links):
- https://github.com/angular/angular/blame/4ca7818ce/aio/content/guide/releases.md#L130
- https://github.com/angular/angular/blame/4ca7818ce/aio/content/guide/updating.md#L72

gkalpak reviewed (dismissed):
:grin:

StephenFluin committed:
docs: remove the release schedule from docs

Removing for now, since the info is stale, and we need to determine how to adjust the schedule due to the unexpected and significant version 9 delay.

IgorMinar reviewed (approved):
please rebase and update the commit message with the justification:

```
Removing for now, since the info is stale, and we need to determine how to adjust the schedule due to the unexpected and significant version 9 delay.
```

IgorMinar commented:
I made the updates + rebased

kara commented:
Should be on 8.2.x now with https://github.com/angular/angular/commit/72c3ebe682ef0a12e9b1f725d85bfb79fd23f4e5
